src/test/java/core/basesyntax/service/parsefileinfo/TransactionParserTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.parsefileinfo;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class TransactionParserTest {
+    private String validString;
+    private final TransactionParser parser = new TransactionParser();
```

Comment:
Creat an init method pls

Reply:
dont think that it's necessary here. Because we use it only in one testCase